transfer.js

```diff
@@ -0,0 +1,63 @@
+const transferForm = document.getElementById("transfer-form");
+const selectBeneficiaryAccountNumberElem = document.getElementById(
+  "beneficiary-account-number"
+);
+const transferAmountElem = document.getElementById("transfer-amount");
+const accountPinElem = document.getElementById("account-pin");
+const currentUser = getUserByAccountNumber(currentUserAccountNumber);
+const allUsers = getAllUsers();
+// populate users in our database into the select field and remove the current user account number
+allUsers.forEach((user) => {
+  for (let i = 0; i < selectBeneficiaryAccountNumberElem.length; i++) {
+    if (
+      selectBeneficiaryAccountNumberElem.options[i].value ==
+      currentUserAccountNumber
+    ) {
+      selectBeneficiaryAccountNumberElem.remove(i);
+    }
+  }
+  const optionElem = document.createElement("option");
+  optionElem.value = user.accountNumber;
+  optionElem.textContent = `${user.accountNumber}`;
+  selectBeneficiaryAccountNumberElem.append(optionElem);
+});
+
+const transfer = () => {
+  // get the previous transaction before this one
+  const currentBalance = getUserCurrentBalance();
+
+  if (accountPinElem.value !== currentUser.accountPin) {
+    alert("Incorrect PIN.");
+    return;
+  }
+  if (transferAmountElem.value > currentBalance) {
+    alert("Insufficient Balance");
+    return;
+  }
+  if (transferAmountElem.value < 0) {
```

Comment:
This check should be `(transferAmountElem.value < 1)`. You can't send 0 Naira to someone.

Reply:
Done